### ISSUE TITLE
Using different loss functions

### ISSUE THREAD

**Issue by rajibchakravorty** [rajibchakravorty/tf_framework#3]
Apparently some loss functions prefer one-hot-label and some just indices - this needs to be configured.
Also some may prefer probabilities, some unscaled logits.
Need to check and should provide a configurable way to do this.

**Comment by rajibchakravorty** [comment]
This is not configurable from config file. Same is done for optimizers.
